code/modules/mob/dead/new_player/sprite_accessories/ears.dm

```diff
@@ -494,3 +494,60 @@
 	icon = 'modular_citadel/icons/mob/32x64_mam_ears.dmi'
 	icon_state = "jackalope"
 	matrixed_sections = MATRIX_RED
+
+/datum/sprite_accessory/ears/human/pichu
+	name = "Pichu"
+	icon = 'modular_splurt/icons/mob/32x64_poke_ears.dmi'
+	icon_state = "pichu"
+	color_src = MATRIXED
+	matrixed_sections = MATRIX_RED_GREEN
+
+/datum/sprite_accessory/ears/mam_ears/pichu
+	name = "Pichu"
+	icon = 'modular_splurt/icons/mob/32x64_poke_ears.dmi'
+	icon_state = "pichu"
+	color_src = MATRIXED
+	matrixed_sections = MATRIX_RED_GREEN
+
+/datum/sprite_accessory/ears/human/pikachu
+	name = "Pikachu"
+	icon = 'modular_splurt/icons/mob/32x64_poke_ears.dmi'
+	icon_state = "pika"
+	color_src = MATRIXED
+	matrixed_sections = MATRIX_RED_GREEN
+
+/datum/sprite_accessory/ears/mam_ears/pikachu
+	name = "Pikachu"
+	icon = 'modular_splurt/icons/mob/32x64_poke_ears.dmi'
+	icon_state = "pika"
+	color_src = MATRIXED
+	matrixed_sections = MATRIX_RED_GREEN
+
+/datum/sprite_accessory/ears/human/bunnypokemon
+	name = "Bunny (Pokemon) (Short)"
+	icon = 'modular_splurt/icons/mob/32x64_poke_ears.dmi'
+	icon_state = "pokebunny"
+	color_src = MATRIXED
+	matrixed_sections = MATRIX_RED_GREEN
+
+/datum/sprite_accessory/ears/mam_ears/bunnypokemon
+	name = "Bunny (Pokemon) (Short)"
+	icon = 'modular_splurt/icons/mob/32x64_poke_ears.dmi'
+	icon_state = "pokebunny"
+	color_src = MATRIXED
+	matrixed_sections = MATRIX_RED_GREEN
+
+/datum/sprite_accessory/ears/human/lopunny
+	name = "Lopunny (big)"
+	icon = 'modular_splurt/icons/mob/32x64_poke_ears.dmi'
+	icon_state = "lopunny"
+	color_src = MATRIXED
+	matrixed_sections = MATRIX_RED_GREEN
+
+/datum/sprite_accessory/ears/mam_ears/lopunny
+	name = "Lopunny (big)"
+	icon = 'modular_splurt/icons/mob/32x64_poke_ears.dmi'
+	icon_state = "lopunny"
+	color_src = MATRIXED
+	matrixed_sections = MATRIX_RED_GREEN
+
```

Comment:
Please put this in a modular file

Reply:
But isn't this already into the modular_splurt folder? i got the same question but i thought it was just fine to leave it there :/

Reply:
could be gentle to tell me _where_ can i move the piece of code?

Reply:
Ah, my bad. seems like i have used the wrong file all this time instead of the one that is actually in the modular-